src/main/java/io/appform/dropwizard/actors/base/UnmanagedPublisher.java

```diff
@@ -88,15 +88,15 @@ public void start() throws Exception {
 
         this.publishChannel = connection.newChannel();
         connection.ensure(queueName + "_SIDELINE", queueName, dlx,
-                connection.rmqOpts());
+                connection.rmqOpts(config.getTtlConfig()));
```

Comment:
If the actor level config is not set, this argument is basically `null` being passed around multiple levels of methods which can be a potential cause of NPEs for any future changes. Something better could be:
`TtlConfig actorConfig = config.getTtlConfig();
if (actorConfig != null) {
 // pass the ttl config argument
else {
 // without the ttl config
}
`

Reply:
Changed and passed actorconfig